experimental/wgit/cli.py

```diff
@@ -5,18 +5,53 @@
 
 import argparse
 
+import experimental.wgit.weigit_api as weigit_api
+
 
 def main(argv=None):
-    desc = "WeiGit checkpoint tracking"
+    desc = "WeiGit: A git-like tool for Model checkpoint tracking"
     parser = argparse.ArgumentParser(description=desc)
 
     # flags
     parser.add_argument("-i", "--init", action="store_true", help="Initialize a weigit repository!")
+    parser.add_argument(
+        "-a",
+        "--add",
+        default="",
+        type=str,
+        metavar="FILE_PATH",
+        help="path to the checkpoint to be added (default: none)",
+    )
+    parser.add_argument("-s", "--status", action="store_true", help="Show the .wgit status")
+    parser.add_argument("-l", "--log", action="store_true", help="Show the log of .wgit")
+    parser.add_argument("-c", "--commit", action="store_true", help="Commit to .wgit")
+    parser.add_argument("-co", "--checkout", action="store_true", help="checkout from a commit")
 
     args = parser.parse_args(argv)
 
     if args.init:
-        print("Hello World, Wgit has been initialized!")
+        weigit = weigit_api.WeiGit()
```

Comment:
basically, there should be a class that represent a "repo". A repo is a dir that contains a .wgit subdir that is already initialized. Other than the init function, the rest of functions should all take an object for a repo. Let's chat in more details offline.